src/app/project/compliance/inspection-detail/inspection-detail.component.ts

```diff
@@ -115,11 +118,26 @@ export class InspectionDetailComponent implements OnInit, OnDestroy {
         this.tableParams.pageSize = 100000;
         this.setRowData();
         this.loading = false;
+        let showError = false;
+
 
         this.assets.forEach(async z => {
           if (z.type === 'photo') {
+            let resource;
             // Show thumb
-            let resource = await this.api.downloadElementThumbnail(this.compliance._id, this.submission._id, z._id);
+            await this.api.downloadElementThumbnail(this.compliance._id, this.submission._id, z._id).then(response => {
```

Comment:
Can remove `await because it is now using `.then()`

Reply:
can do!